ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -274,6 +289,10 @@ export default class StatusDropdown extends React.Component {
                     Mark as fixed
                   </DropdownItem>
                 )}
+
+                <DropdownItem onClick={() => this.toggle('showTagsModal')}>
```

Comment:
I think this needs to have a `tag=a` attribute per our docs on accessibility changes. The `DropdownItems` above might also need to have that added as well.